pom.xml

```diff
@@ -80,7 +80,7 @@
         <slf4j.version>1.7.36</slf4j.version>
         <jaxb.api.version>2.3.3</jaxb.api.version>
         <jaxb.impl.version>2.3.6</jaxb.impl.version>
-        <jackson.version>2.13.2</jackson.version>
+        <jackson.version>2.13.2.1</jackson.version>
```

Comment:
alleen van databind is deze versie er, niet van core...
`Could not find artifact com.fasterxml.jackson.core:jackson-core:jar:2.13.2.1`